src/simple_http.rs

```diff
@@ -362,6 +395,21 @@ impl Builder {
         self
     }
 
+    /// Add proxy address to the transport for SOCKS5 proxy
+    #[cfg(feature = "proxy")]
+    pub fn proxy_addr<S: AsRef<str>>(mut self, proxy_addr: S) -> Result<Self, Error> {
+        self.tp.proxy_addr = check_url(proxy_addr.as_ref())?;
+        Ok(self)
+    }
+
+    /// Add optional proxy authentication as ('username', 'password')
+    #[cfg(feature = "proxy")]
+    pub fn proxy_auth<S: AsRef<str>>(mut self, proxy_auth: Option<(S, S)>) -> Self {
```

Comment:
Its not immediately clear to me what the point of the `Option` is? Is the point so that `proxy_auth(None)` can be used to clear the proxy auth? If so perhaps document that. If that is not the use then perhaps we should use args `(user: S, pass: S)`.

Reply:
You are right. Having `Option<>` doesn't make much sense here.. Updated to your second suggestion..